src/device/scrollCursorIntoView.ts

```diff
@@ -24,6 +24,7 @@ const scrollIntoViewIfNeeded = (el: Element | null | undefined) => {
   // toolbar element is not present when distractionFreeTyping is activated
   const toolbarRect = document.getElementById('toolbar')?.getBoundingClientRect()
   const toolbarBottom = toolbarRect ? toolbarRect.bottom : 0
+  const navbarRect = document.querySelector('.nav.nav-bottom')?.getBoundingClientRect()
```

Comment:
Let's query by id, data-attribute, or aria-label so this behavior is not tightly coupled to the CSS class names.